composer.json

```diff
@@ -18,22 +18,12 @@
     "issues": "https://github.com/stayallive/wp-sentry/issues"
   },
   "autoload": {
-    "psr-0": {
-      "WP_Sentry_": "."
-    }
+    "classmap": [
+      "src/"
+    ]
   },
   "require": {
     "php": ">=5.4.0",
-    "composer/installers": "~1.0",
     "sentry/sentry": "^1.7.1"
-  },
-  "require-dev": {
-    "dealerdirect/phpcodesniffer-composer-installer": "^0.4.3",
-    "wp-coding-standards/wpcs": "^0.13.1"
-  },
-  "scripts": {
-    "test": [
```

Comment:
Why do you remove test scripts? 🤔

Reply:
Because phpcs is not used (by my / automated) and the current setup had 200+ errors 👎